### PR TITLE
Fix LGTM alerts and flake8 warnings

### PR DESCRIPTION
**Pull request by jsf9k**
I only fixed flake8 warnings that can be easily fixed with only whitespace changes.

I also took the opportunity to update the example extract config file for the changes made in #19.

### CODE REVIEW

**Comment by jsf9k** [comment]
This pull request **fixes 2 alerts** when merging 3f5c17c0d38ef738667b64daa439c3960fdc6e59 into 2ebd13c127649c73eac59dbf3824cb1849b5e77a - [view on LGTM.com](https://lgtm.com/projects/g/cisagov/cyhy-feeds/rev/pr-1934c6d8296521fbce74bc6d209160f4bc367f23)

**fixed alerts:**

* 1 for Unused local variable
* 1 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: aws_jobs/cyhy-data-retriever.py
Comment (jsf9k) [inline aws_jobs/cyhy-data-retriever.py:48]: I removed this because it isn't used anywhere.  Can it be removed from the example config as well?

**Review by felddy — APPROVED** [review]
Seems like it the example file could also have the unused param removed.  I think @KyleEvers should have the final say.

File: aws_jobs/cyhy-data-retriever.py
Comment (felddy) [reply aws_jobs/cyhy-data-retriever.py:48]: I think so, but get the go-ahead from @KyleEvers

**Comment by jsf9k** [comment]
No problem.  I was going to wait for @KyleEvers' review anyway.

**Comment by KyleEvers** [comment]
I tested these changes and it works fine. 💯

**Pull request merged**
